### ISSUE TITLE
SendCommand java samples not working fully

### ISSUE THREAD

**Issue by Kiwa21** [ugcs/ugcs-java-sdk#5]
Hey,

I just built the ugcs-java-sdk repo with maven and I can run the SendCommand sample with java.
However, differents commands seems to work unproperly : 

- SendCommand -c waypoint : when giving latitude / longitude / altitude and "EMU-COPTER-17" as arguments, I can see the emulation copter moves in the regular client but it does not go to the specified waypoint..?

- SendCommand -c return_to_home works but final alt is mixing AGL and AMSL.. My home location is AGL 0m and AMSL 329m and copter went to the right lat/long but with AGL 329m..

Thanks for your review !

**Comment by Kiwa21** [comment]
Actually, same thing is happening with the C# SDK

https://github.com/ugcs/ugcs-dotnet-sdk/blob/master/UGCS.Example/UGCS.Console/Program.cs

When using only the Click&Go function, copter start moving, but not at the right location ? Seems to never stop but I don't know where it is going !

**Comment by episarenko** [comment]
Hi,

target position for the waypoint command should be specified in radians not in degrees. If everything is correct could you please post the full command string here (including args).

Regarding the wrong home altitude - I'll check it and reply shortly.

**Comment by episarenko** [comment]
Yes there is an issue with the wrong target altitude in the RTH. It only affects ugcs emulator drones and will be fixed in 2.12. I hope it will not cause much trouble for you.
